packages/w3up-client/README.md

```diff
@@ -47,46 +47,39 @@ npm install @web3-storage/w3up-client
 
-When invoking storage capabilities, the Space ID is the "resource" portion of the capability, while the ability is an action like `store/add` or `store/remove`.
+#### Space
 
-Both Agents and Spaces are identified using _DIDs_, or Decentralized Identity Documents. DIDs are a [W3C specification](https://www.w3.org/TR/did-core/) for verifiable identities in decentralized systems. There are several DID "methods," but the ones most commonly used by w3up are [`did:key`](https://w3c-ccg.github.io/did-method-key/), which includes a public key directly in the DID string. Agents and Spaces both use `did:key` URI strings as their primary identifiers. The other DID method used by w3up is [`did:web`](https://w3c-ccg.github.io/did-method-web/), which is used to identify the service providers.
+When you upload data to w3up, your uploads are linked to a unique _Space_ that acts as a "namespace" for the data you upload. Each Space corresponds to a _DID_, or [Decentralized Identity Document](https://www.w3.org/TR/did-core/). Specifically, a Space corresponds to a [`did:key`](https://w3c-ccg.github.io/did-method-key/), which is a public key with a corresponding private signing key. 
 
-Agents and Spaces are both generated by `w3up-client` on the user's local machine. Before they can be used for storage, the user will need to [register the space](#creating-and-registering-spaces) by confirming their email address. Once registered, a Space can be used to [upload files and directories](#uploading-data).
+When creating a Space using `w3up-client`, it generates this private key and `did:key` for you locally. To use web3.storage, you then register a Space by associating it with your email address. From there, when invoking storage capabilities with web3.storage, the Space `did:key` is the "resource" portion of the capability, while the ability is an action like `store/add` or `store/remove`. (A Space registered with web3.storage is imperfectly analogous to an "account" with web3.storage.)
 
-### Basic usage
+Under the hood in the email registration process, your Space delegates to your email address the ability to call any capability; this delegation is stored by web3.storage. So if you ever need access to your Space in the future from any device, web3.storage allows you to "sign in" by verifying your email, so you don't need keep your Space private key around - no complex local key management needed! (`w3up-client` actually throws this private away for safety reasons after registration.) More on this "sign in" process is detailed in the next section on Agents.
```

Comment:
"Under the hood in the email registration process, your Space delegates the capabilities needed to use `w3up` to your email address and this delegation is stored by web3.storage. If you need access to your Space in the future from another device, web3.storage allows you to reclaim those capabilities the same way you would reset a password in other services - using an email verification process. This means you don't need to store or manage Space private keys to use w3up - just create a new space, register it with w3up and use it from as many devices as you like."